src/commands/moderation/untimeout.ts

```diff
@@ -67,7 +67,7 @@ const failedUnTimeoutEmbed = (user: User) => (makeEmbed({
 
 export const untimeout: CommandDefinition = {
     name: ['untimeout', 'removetimeout'],
-    requiredPermissions: ['BanMembers'],
+    requirements: { permissions: ['BanMembers'] },
```

Comment:
Not a comment on the PR, but @benw202 : Does this make sense, should we take the opportunity to change this to ADMIN & MODERATOR roles?